Rename advisors (#10)
percona-platform/portal-doc: docs/april-2022.md

PMM instances that are already connected to Percona Platform will not be automatically terminated with this update. However, we recommend that you disconnect all existing PMM instances and reconnect them using the new access-token authorization.

To check out the new token-based authentication, go to **PMM > Settings > Percona Platform** and click **Get token from Percona Platform** next to the **Percona Platform Access Token** field.

For more information, see [Connect your PMM server](connect-pmm.md).

### Synchronized customer information with PMM 
After connecting PMM to Percona Platform as a Percona customer, PMM now reveals two new tabs on the main menu: **Entitlements** and **Support tickets**. 
 
Percona Platform populates these tabs with information related to their Percona Platform entitlements and Customer Support cases.

For more information, see [Integrate PMM with Percona Platform](https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/integrate-platform.html) in the PMM online Help.

### New Paid tier and special PMM Advisor checks for customers

> When you connect your PMM to Pecona Platform with an account linked to Percona Customer Portal, you now also have access to Paid Advisor checks in addition to Registered checks. 

Reviewer: ```suggestion
When you connect your PMM to Pecona Platform with an account linked to Percona Customer Portal, you will get access to Paid Advisor checks in addition to Registered checks. 
```

Paid Advisor checks offer more advanced information about the health of your databases.

Here are the new list of checks introduced with this update:

**New MySQL Advisor Checks**

- Validate Password Policy
- Local Option of Load Data Enabled
- Not Verifying Checksum
- SQL Processing Not Multi-Threaded
  
**New MongoDB Advisor Checks**

- Check the Replica Set Topology
- Check Active/Available Connection Ratio

See the full list of database checks in the [Advisor checks for PMM](checks.md) topic. 
## Enhanced User Experience to boost your productivity
We’ve also tweaked the structure of the main menu to improve the navigation when moving from one flow to another. 

The links to managing existing organizations and connected PMM instances are no longer hidden inside the **Get Started** tab. Instead, they are exposed as dedicated tabs on the main menu: **Organization** and **PMM Instances**. 

> In addition, the **Get Started** tab has been merged into the **Dasboard** tab, which now hosts all the tools for creating your organization and connecting PMM instances to it. 

Reviewer: ```suggestion
In addition, the **Get Started** tab has been merged into the **Dashboard** tab, which now hosts all the tools for creating your organization and connecting PMM instances to it. 
```

You'll also find here information related to your account, as well as links to contact Percona for support, consulting, or managed services. 

> You may also want to know that the **Resources** menu now also includes a link to this online Help.

Reviewer: ```suggestion
The **Resources** menu now also includes a link to this online Help, so that you can reach this space any time from within Platform Portal.
```
